content/en/tracing/setup_overview/setup/dotnet-framework.md

```diff
@@ -232,45 +231,44 @@ To configure the Tracer using an `app.config` or `web.config` file, use the `<ap
 
 {{% tab "JSON file" %}}
 
-To configure the Tracer using a JSON file, create `datadog.json` in the instrumented application's directory. The root JSON object must be a hash with a key-value pair for each setting. For example:
+To configure the Tracer using a JSON file, create `datadog.json` in the instrumented application's directory. The root JSON object must be an object with a key-value pair for each setting. For example:
 
 ```json
 {
-    "DD_TRACE_AGENT_URL": "http://localhost:8126",
-    "DD_ENV": "prod",
-    "DD_SERVICE": "MyService",
-    "DD_VERSION": "abc123",
+  "DD_TRACE_AGENT_URL": "http://localhost:8126",
+  "DD_ENV": "prod",
+  "DD_SERVICE": "MyService",
+  "DD_VERSION": "abc123",
 }
 ```
 
 {{% /tab %}}
 
 {{< /tabs >}}
 
-## Configuration settings
+### Configuration settings
 
 Using the methods described above, customize your tracing configuration with the following variables. Use the environment variable name (for example, `DD_TRACE_AGENT_URL`) when setting environment variables or configuration files. Use the TracerSettings property (for example, `AgentUri`) when changing settings in code.
 
-### Unified Service Tagging
-
-To use [Unified Service Tagging][7], configure the following settings for your services.
+#### Unified Service Tagging
 
+To use [Unified Service Tagging][7], configure the following settings for your services:
 
 `DD_ENV`
 : **TracerSettings property**: `Environment`<br>
-If specified, adds the `env` tag with the specified value to all generated spans. Added in version 1.17.0
+If specified, adds the `env` tag with the specified value to all generated spans. Added in version 1.17.0.
 
 `DD_SERVICE`
 : **TracerSettings property**: `ServiceName`<br>
-If specified, sets the service name. Otherwise, the .NET Tracer tries to determine service name automatically from application name (e.g. IIS application name, process entry assembly, or process name). Added in version 1.17.0
+If specified, sets the service name. Otherwise, the .NET Tracer tries to determine service name automatically from application name (e.g. IIS application name, process entry assembly, or process name). Added in version 1.17.0.
```

Comment:
```suggestion
If specified, sets the service name. Otherwise, the .NET Tracer tries to determine service name automatically from application name (IIS application name, process entry assembly, or process name). Added in version 1.17.0.
```